lib/loader.js

```diff
@@ -353,9 +353,11 @@ module.exports = function (content) {
   '  }\n' +
   '})\n'
 
-  // support to register static styles
-  exports += 'if (typeof __register_static_styles__ === "function") {\n' +
-  '  __register_static_styles__(__vue_options__._scopeId, __vue_styles__)\n' +
+  // support to register style sheets
+  exports += 'if (typeof weex === "object" && weex && weex.document) {\n' +
```

Comment:
I thought the simpler judged code is `weex && weex.document` that is enough to avoid all problems.

Reply:
What if `weex` doesn't exist or equals `null` in the context? I think this validation is much safer.

Reply:
In javascript, while `null` and `undefined` wrapped in the `if` block, they are all considered as `false`, or you can also write like this `typeof weex === "object" && weex.document`, the `weex` is unnecessary.

Reply:
I just think this logic is a bit redundant！